react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -15,6 +15,7 @@ export const layersDefaultProps: Record<string, unknown> = {
             step: 0,
         },
         rotDeg: 0,
+        colorMapName: "Rainbow",
```

Comment:
default colorMapRange and valueRange are not defined properly in ColorMapLayer and Hillshading2DLayer.
Need to be { type: "array", value: [0, 1]}